streams/readable-byte-streams/general.any.js

```diff
@@ -957,7 +957,8 @@ promise_test(() => {
     assert_equals(view.byteOffset, 0, 'byteOffset');
     assert_equals(view.byteLength, 2, 'byteLength');
 
-    assert_equals(view[0], 0x0201);
+    const data_view = new DataView(view.buffer, view.byteOffset, view.byteLength);
```

Comment:
Nit: camelcase variable names

Reply:
Woops, brainfart. 😅 I was first considering adding a helper function for this, and we usually use `snake_case` for those. Fix incoming!